module-system/sov-modules-core/src/common/gas.rs

```diff
@@ -80,9 +108,21 @@ where
         self.remaining_funds
     }
 
+    /// Returns the total gas incurred.
+    pub const fn gas_total(&self) -> &GU {
```

Comment:
nit: The name "gas_used" might be slightly more clear. WDYT?

Reply:
Makes sense; fixed!